tools/detect_targets.py

```diff
@@ -92,6 +99,32 @@ def main():
         traceback.print_exc(file=sys.stdout)
         print "[ERROR] %s" % str(exc)
         sys.exit(1)
+        
+def get_interface_version(mount_point):
+    """ Function returns interface version from the target mounted on the specified mount point
+    
+        mount_point can be acquired via the following:
+            muts = get_autodetected_MUTS_list()
+            for mut in muts.values():
+                mount_point = mut['disk']
+                    
+        @param mount_point Name of disk where platform is connected to host machine.
+    """
+    if get_module_avail('mbed_lstools'):
+        mbeds = mbed_lstools.create()
+        details_txt = mbeds.get_details_txt(mount_point)
+        
+        if (details_txt is None):
+            details_txt = {}
```

Comment:
Could you just 
```python
return "unkown"
```
here?

Reply:
Makes sense for a unit test, I hadn't realised how many variations of details.txt there could be until now. I'll work on this in the coming few days.